visualization/FairnessDashboard/src/AccuracyMetrics.ts

```diff
@@ -181,5 +181,20 @@ export const AccuracyOptions: {[key: string]: IAccuracyOption} = {
         isMinimization: false,
         isPercentage: true,
         userVisible: true
-    }
+    },
+    "f1_score": {
+        key: "f1_score",
+        title: localization.Metrics.f1Score,
+        description: localization.Metrics.f1ScoreDescription,
+        isMinimization: false,
+        isPercentage: false,
+        userVisible: true
+    },
+    "log_loss": {
+        key: "log_loss",
+        title: localization.Metrics.logLoss,
+        description: localization.loremIpsum,
```

Comment:
@romanlutz  or @MiroDudik  we definitely don't want placeholder description. Though this is currently set to not be user visible (is that intentional?)

Reply:
I just copied from other metrics here. I was similarly surprised to see this (hence my message to you yesterday)

Reply:
This isn't currently visible. If we want the ones currently with loremIpsum to be visible we need to write all the descriptions. Not doing that in this PR